src/Host/Client/Impl/Session/RSessionProvider.cs

```diff
@@ -197,8 +197,12 @@ public class RSessionProvider : IRSessionProvider {
                 BrokerChanging?.Invoke(this, EventArgs.Empty);
                 await SwitchBrokerAsync(cancellationToken, oldBroker);
                 oldBroker.Dispose();
-                PrintBrokerInformation();
-            } catch(Exception ex) {
+
+                if (brokerClient.IsRemote) {
+                    _callback.WriteConsole(Environment.NewLine + Resources.Connected + Environment.NewLine);
+                    PrintBrokerInformation();
```

Comment:
Generally user knows local machine parameters, so less clutter. They can request it via Session | Host Information command.